cli/packages/prisma-cli/package.json

```diff
@@ -48,13 +50,14 @@
   "scripts": {
     "build": "rimraf dist && tsc -d && chmod +x dist/index.js",
     "lint": "tslint src/**/*.ts",
-    "publish-homebrew": "dotenv -e scripts/.env ts-node ./scripts/release-brew.ts",
-    "make-binary": "npm run build && npm run package",
+    "publish-binaries": "dotenv -e scripts/.env ts-node ./scripts/release-binaries.ts",
+    "binary-osx": "npm run build && npm run package",
+    "binary-linux": "npm run build && npm run package-linux",
     "precommit": "lint-staged",
     "prettier": "prettier --single-quote --no-semi --trailing-comma all --write '*.ts' 'src/**/*.ts'",
     "prepublishOnly": "npm run lint && npm run build",
-    "package": "./scripts/cp-assets.sh && pkg package.json --targets node10-macos-x64",
-    "package-linux": "./scripts/cp-assets.sh && pkg package.json --targets node10-linux-x64"
+    "package": "rm -rf ./prisma && ./scripts/cp-assets.sh && pkg package.json --targets node10-macos-x64",
```

Comment:
Thanks for adding the remove script, we can also do a "clean up" after publishing operation by removing tar files once the upload is done while packaging.